### DISCUSSION TITLE
Improve visual indication of difference between open and close menu

### DISCUSSION THREAD

**Discussion by techfg** [withastro/starlight#2699] [Feature Requests]
### What version of `starlight` are you using?

0.30.1

### What is your idea?

Currently, the visual difference between menu open & close is accomplished by the use of color.

Would like to see this changed to provide more clarity on what clicking the button will do by using an "X" (the `close` icon) when the menu is open, leaving all the styling identical between opened and closed states.

**Dark Theme**
| | Open | Closed |
| -------- | ------- | ------- | 
| Current | ![image](https://github.com/user-attachments/assets/81906e52-a049-4f00-b2a1-e8c6bd02899e) | ![image](https://github.com/user-attachments/assets/5b52fe88-44c2-436c-918d-86b70aa60f5b) |
| Proposed | ![image](https://github.com/user-attachments/assets/4d0c1614-bb24-4682-bf55-766c604ae544) | ![image](https://github.com/user-attachments/assets/191d4150-b7de-407b-a240-4ef6d3b8ba5f) |

**Light Theme**
| | Open | Closed |
| -------- | ------- | ------- | 
| Current | ![image](https://github.com/user-attachments/assets/fef0008c-40fa-46cd-b322-2af66f6dd857) | ![image](https://github.com/user-attachments/assets/cfcfa2b2-5663-41a6-889d-296a7212924b) |
| Proposed | ![image](https://github.com/user-attachments/assets/c922aba6-aab7-44e5-956c-0eebcff61615) | ![image](https://github.com/user-attachments/assets/d13c8fc0-1009-4ede-b05a-c4b4ad3e4287) |

### Why is this feature necessary?

The concern with the current behavior is that by darkening the button when the menu is opened is that its more akin to being disabled then active.  Additionally, based on color scheme/theme, the difference could be very subtle and difficult to discern.

### Do you have examples of this feature in other projects?

- https://vitepress.dev/guide/what-is-vitepress
- https://docusaurus.io/docs

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
The suggested change was released in [Starlight 0.33.0](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.33.0) :tada:

Thanks for your amazing work.
